### PR TITLE
Update _params.py

### PR DESCRIPTION
**Pull request by bjqian**
remove hard restriction on sku. Leave this to server side for future Premium_P1, Standard_S2

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### Related command

az webpubsub

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repository and upgrade the version in the pull request but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
webpubsub

File: src/webpubsub/azext_webpubsub/_params.py [hunk 1]
```diff
@@ -33,12 +32,12 @@ def load_arguments(self, _):
         c.argument('webpubsub_name', webpubsub_name_type, options_list=['--name', '-n'])
 
     with self.argument_context('webpubsub create') as c:
-        c.argument('sku', arg_type=get_enum_type(SKU_TYPE), help='The sku name of the signalr service.')
-        c.argument('unit_count', help='The number of signalr service unit count', type=int)
+        c.argument('sku', help='The sku name of the webpubsub service. E.g. Standard_S1')
```
Comment (kairu-ms) [inline src/webpubsub/azext_webpubsub/_params.py:35]: why diable the choice options of sku argument?

Comment (bjqian) [reply src/webpubsub/azext_webpubsub/_params.py:35]: We have SKUs that are internal only. Curretly we don't hope customers outside Microsoft to see those SKUs as they are not allowed to create them. But we hope az cli is available for our internal customers. So we need to remove this restriction from client side and leave it to server side.

Comment (kairu-ms) [reply src/webpubsub/azext_webpubsub/_params.py:35]: The sku parameter is required for `az webpubsub create`. Did you mean the customers outside Microsoft are not supposed to run the create command?

Comment (zackliu) [reply src/webpubsub/azext_webpubsub/_params.py:35]: I think we can put all `sku` available for public in `help`. But as it's a server-side parameter, I think it doesn't make sense to make restrictions in cli side. We can pass `sku` parameter to RP side for checking.

Comment (kairu-ms) [reply src/webpubsub/azext_webpubsub/_params.py:35]: > I think we can put all `sku` available for public in `help`. But as it's a server-side parameter, I think it doesn't make sense to make restrictions in cli side. We can pass `sku` parameter to RP side for checking.

I totally agree

Comment (bjqian) [reply src/webpubsub/azext_webpubsub/_params.py]: Fixed

**Review by kairu-ms — APPROVED** [review]

**Comment by kairu-ms** [comment]
@bjqian If you want to release a new version, please create a new PR with extension version updated.

**Pull request merged**